cloudwatchlogs/cloudwatchlogs_lambda.js

```diff
@@ -8,7 +8,7 @@ exports.handler = function(event, context) {
 		///////////////////////////////////////////////////////////////////////////////////////////////////////////
 		var options = {
 			'hostname': 'collectors.sumologic.com',
-			'path': 'https://collectors.sumologic.com/receiver/v1/http/<XXX>',
+			'path': 'receiver/v1/http/<XXX>',
```

Comment:
Here is an example of what I tested: 
var options = { 'protocol':'https:',
                    'hostname': 'endpoint1.collection.sumologic.com',
                    'path': '/receiver/v1/http/ZaVnC4dhaV1eGgsrsk0rInUgrTF6ZVa7AxTxe0wxyY-1iFn_uQKtSPeAKyppmlyUloXimG092QQyxemDWVRQr7pPYma45NXIIhHsXcfk0m6WnJI3T8YkTA==',
                    'method': 'POST'
    };